### PR TITLE
Change model datetime from SystemTime to chrono DateTime

### PR DESCRIPTION
**Pull request by code-mm**
This PR changes the used data type for datetime in the db models. Change is required for the implementation of tests.

### CODE REVIEW

File: identity/tests/sample_data.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use std::time::{Duration, SystemTime};
+use chrono::Utc;
```
Comment (Urhengulas) [inline identity/tests/sample_data.rs:1]: Why not also import Duration here?

File: identity/tests/sample_data.rs [hunk 2]
```diff
@@ -22,7 +22,7 @@ pub fn users() -> Vec<UserAddUpdate> {
             family_name: "Kerr".into(),
             picture: "https://example.com/avatar.jpg".into(),
             oauth_access_token: "access_token".into(),
-            oauth_access_token_valid: SystemTime::now() + Duration::from_secs(3600),
+            oauth_access_token_valid: Utc::now().naive_utc() + chrono::Duration::seconds(3600),
```
Comment (Urhengulas) [inline identity/tests/sample_data.rs:25]: Is there a `Duration::minutes` or `Duration::hours`?

**Review by Urhengulas — CHANGES_REQUESTED** [review]

**Comment by code-mm** [comment]
There was a mistake when I rebased this. It is fixed now. I need a fixed datetime there, otherwise I can not properly assert it.

File: identity/Cargo.toml [hunk 3]
```diff
@@ -14,7 +14,8 @@ test = false
 doc = false
 
 [dependencies]
-diesel = { version = "1.4", features = ["postgres", "r2d2"] }
+chrono = { version = "0.4", features = ["serde"] }
+diesel = { version = "1.4", features = ["postgres", "r2d2", "chrono"] }
```
Comment (Urhengulas) [inline identity/Cargo.toml:18]:
```suggestion
diesel = { version = "1.4", features = [ "chrono", "postgres", "r2d2"] }
```

File: identity/tests/sample_data.rs [hunk 4]
```diff
@@ -11,7 +11,7 @@ pub fn users() -> Vec<UserAddUpdate> {
             family_name: "Doe".into(),
             picture: "https://example.com/avatar.jpg".into(),
             oauth_access_token: "access_token".into(),
-            oauth_access_token_valid: SystemTime::now() + Duration::from_secs(3600),
+            oauth_access_token_valid: NaiveDate::from_ymd(2020, 12, 31).and_hms(0, 0, 0),
```
Comment (Urhengulas) [inline identity/tests/sample_data.rs:14]:
```suggestion
            oauth_access_token_valid: NaiveDate::from_ymd(2020, 12, 31),
```
Can't you leave the `.and_hms(0, 0, 0)` away?

**Review by Urhengulas — DISMISSED** [review]

File: identity/tests/sample_data.rs (hunk 4 above)
Comment (code-mm) [reply identity/tests/sample_data.rs:14]: In theory yes, but I prefer it explicit here and also I want to adapt the tests later to more complex scenarios.

Comment (Urhengulas) [reply identity/tests/sample_data.rs:14]: 👍🏾

**Pull request merged**